### PR TITLE
Reenable clustering tests

### PR DESCRIPTION
**Pull request by gerzse**
Uncomment clustering tests and adapt the code in the minimum possible way to make these tests run.

### CODE REVIEW

File: src/test/java/redis/clients/jedis/HostAndPorts.java [hunk 1]
```diff
@@ -34,6 +35,10 @@ public final class HostAndPorts {
     clusterHostAndPortList.add(new HostAndPort("localhost", 7382));
     clusterHostAndPortList.add(new HostAndPort("localhost", 7383));
     clusterHostAndPortList.add(new HostAndPort("localhost", 7384));
+
+    stableClusterHostAndPortList.add(new HostAndPort("localhost", 10379));
+    stableClusterHostAndPortList.add(new HostAndPort("localhost", 10380));
+    stableClusterHostAndPortList.add(new HostAndPort("localhost", 10381));
```
Comment (sazzad16) [inline src/test/java/redis/clients/jedis/HostAndPorts.java:41]: I'd prefer to keep these in 7000 series; i.e. `73XY` or `7X79-81`.

**Review by sazzad16 — CHANGES_REQUESTED** [review]
Some nit-picky comments added. Also, can we not change the TestBase classes or not use hashtags `{}` in non-cluster mode?

**Comment by sazzad16** [comment]
Resolves #2774

File: src/test/java/redis/clients/jedis/HostAndPorts.java (hunk 1 above)
Comment (gerzse) [reply src/test/java/redis/clients/jedis/HostAndPorts.java]: Moved to 7479-7481.

**Comment by gerzse** [comment]
> Some nit-picky comments added. Also, can we not change the TestBase classes or not use hashtags `{}` in non-cluster mode?

The TestBase classes already use that trick, that's how I found it. Initially I wanted to compute keys that hit the same hashslot, but this felt cleaner.

If you think, we can get rid of the hashtags, but maybe we can do it in all places, in a separate PR?

**Comment by sazzad16** [comment]
A quick idea only for this PR, Keep the `...TestBase` classes untouched and override the necessary methods in `Cluster...Test` classes.

**Comment by gerzse** [comment]
> A quick idea only for this PR, Keep the `...TestBase` classes untouched and override the necessary methods in `Cluster...Test` classes.

Changed as you suggested.

**Review by sazzad16 — APPROVED** [review]

**Pull request merged**